asf_welsh_energy_consultation/pipeline/process_data.py

```diff
@@ -277,6 +278,36 @@ def get_mcs_retrofits():
     return mcs_retrofits
 
 
+def generate_hex_counts(wales_df, pc_df):
+    """
+    Merges two dataframes on 'postcode' and generates pandas.DataFrame containing information on % of properties with
+    heat pumps installed in each Hex 3 partition (https://h3geo.org/docs/) in Wales
```

Comment:
Instead of "two dataframes" i'd say "Merges processed EPC data for Wales with a dataframe
```suggestion
    Merges EPC data for Wales with location information and generates a dataframe containing information on % of properties with
    heat pumps installed in each Hex 3 partition (https://h3geo.org/docs/) in Wales.
```